engine/build.gradle

```diff
@@ -59,6 +59,7 @@ dependencies {
     api group: 'net.sf.trove4j', name: 'trove4j', version: '3.0.3'
     implementation group: 'io.netty', name: 'netty-all', version: '4.1.53.Final'
     implementation group: 'com.google.protobuf', name: 'protobuf-java', version: '2.6.1'
+    implementation 'org.lz4:lz4-java:1.8.0'
```

Comment:
Please use the same syntax/format for adding this dependency as for the dependencies around it :wink: